skimage/morphology/convex_hull.py

```diff
@@ -17,6 +19,23 @@ def _offsets_diamond(ndim):
         offsets[vertex, axis] = offset
     return offsets
 
+def _apply_partial_offsets(img, coords, offsets):
+    """
+    Apply the offsets only to the non-edge pixels, along with the trivial zero-offset.
+    """
```

Comment:
This function is a bit hard to read, could you please add a minimal example here in the docstring?